### ISSUE TITLE
Cannot find translation when using Universal

### ISSUE THREAD

**Issue by schmorell** [ngneat/transloco#197]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[x] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

I have tried to set up a Angular App with Angular Universal and transloco, basically with these steps:
1. npx @angular/cli new transloco-ssr
2. ng add @nguniversal/express-engine --clientProject transloco-ssr
3. ng add @ngneat/transloco (adjusting the port on prod environment)
4. adding a key to the translation file
5. use the key in the emplate

Serving the App with ng s works as expected, but when lauching the App with SSR the translations are not found: Missing translation for 'en.key'

## Expected behavior

I would expect for the that it also works out of the box with SSR

## Minimal reproduction of the problem with instructions

Basically following the steps from this blog: https://dev.to/shaharkazaz/creating-search-engine-friendly-internationalized-apps-with-angular-universal-and-transloco-m02

## Environment

<pre><code>
Angular version: 8.2.14
Universal version: 8.2.6
transloco version: 2.12

Browser:
- [x] Chrome (desktop) version latest
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [x] Firefox version latest
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [x] Edge version latest

**Comment by shaharkazaz** [comment]
@schmorell I'll take a look at it and get back to you 🙂

**Comment by itayod** [comment]
There is a problem with `provideTranslocoConfig` function, please change your code to use `translocoConfig` function instead: 
change this:
```ts
provideTranslocoConfig({
      prodMode: environment.production,
      availableLangs: [{ id: 'en', label: 'English' }, { id: 'es', label: 'Spanish' }],
      reRenderOnLangChange: true,
      fallbackLang: 'es',
      defaultLang: 'en',
      missingHandler: {
      useFallbackTranslation: false
    }
})
```

```ts
{
      provide: TRANSLOCO_CONFIG,
      useValue: translocoConfig({
        prodMode: environment.production,
        availableLangs: [{ id: 'en', label: 'English' }, { id: 'es', label: 'Spanish' }],
        reRenderOnLangChange: true,
        fallbackLang: 'es',
        defaultLang: 'en',
        missingHandler: {
          useFallbackTranslation: false
        }
    })
}
```

**Comment by schmorell** [comment]
Yes, working now. Many thanks for the quick fix. Great library, you are doing an awesome job. Thanks!
